test/integration/tox.ini

```diff
@@ -0,0 +1,14 @@
+[tox]
```

Comment:
@rhcarvalho OK, done! Testing at https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible_tox/61/ but local tests went fine so I expect it to work. Travis already passed.